test/libsolidity/smtCheckerTests/literalSuffixes/overflow_function_single_argument.sol

```diff
@@ -0,0 +1,16 @@
+function intSuffix(int8 x) pure suffix returns (int8) { return x + 1; }
+function uintSuffix(uint y) pure suffix returns (uint) { return y - 1; }
+
+contract C {
+    function overflow() public pure {
+        127 intSuffix;
+    }
+
+    function underflow() public pure {
+        0 uintSuffix;
+    }
+}
+// ----
+// Warning 4984: (63-68): CHC: Overflow (resulting value larger than 127) happens here.
+// Warning 3944: (136-141): CHC: Underflow (resulting value less than 0) happens here.
+// Info 1391: CHC: 1 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
```

Comment:
This is probably the underflow check for `intSuffix`, but we should verify.

Reply:
How can one do that? Some warnings mention an option `show proved safe`, but I haven't figured out how to enable that via `isoltest`.

Reply:
Via `isoltest` that won't work because the settings are specific to the SMTCheckerTests. You'd need to run `solc` itself with `--model-checker-engine all --model-checker-show-proved-targets` or smth like that

Reply:
Thanks! 
The option is  `--model-checker-show-proved-safe`.